frontend/src/components/pages/your_clubs/ClubList.tsx

```diff
@@ -64,6 +71,24 @@ interface ClubListProps {
 export function ClubList(props: ClubListProps) {
   const classes = useStyles();
   const clubsToDisplay = props.clubsToDisplay;
+  const [leaveClubAlertOpen, setLeaveAlertOpen] =
+    React.useState<boolean>(/** closed */false);
```

Comment:
Super minor nit: Inline comments use `/*` instead of `/**`. The one with two asterisks is JSDoc syntax that is only used for function and class comments.

Reply:
Thanks for the clarification!